client/src/components/LandingNavbar/LandingNavbar.tsx

```diff
@@ -1,42 +1,118 @@
-          </div>
-          <Typography className={classes.sitterPage}>
-            <Link to="#" className={` ${pathname === '/' ? classes.sitterLink : classes.otherSitterLink}`}>
-              BECOME A SITTER
-            </Link>
-          </Typography>
-          <div className={classes.login}>
-            <Button
-              component={Link}
-              to="/login"
-              variant="outlined"
-              color="inherit"
-              className={` ${pathname === '/' ? classes.loginBtn : classes.otherLoginBtn}`}
-            >
-              LOGIN
+  const [isDrawerOpen, setIsDrawerOpen] = useState(false);
+  const isMobileView = useMediaQuery('(max-width:600px)');
+
+  const displayDesktop = (
+    <Toolbar className={toolbar}>
+      <Link to="/">
+        <img src={Logo} className={logo} alt="logo" />
+      </Link>
+      <div className={navbarDesktop}>
+        <MenuList className={links}>
+          <MenuItem className={classes.menuItem}>
+            <Typography className={classes.sitterPage}>
+              <Link to="#" className={` ${pathname === '/' ? classes.sitterLink : classes.otherSitterLink}`}>
+                BECOME A SITTER
+              </Link>
+            </Typography>
+          </MenuItem>
+          <MenuItem className={classes.menuItem}>
+            <div className={classes.login}>
+              <Button
+                component={Link}
+                to="/login"
+                variant="outlined"
+                color="inherit"
+                className={` ${pathname === '/' ? classes.loginBtn : classes.otherLoginBtn}`}
+              >
+                LOGIN
+              </Button>
+            </div>
+          </MenuItem>
+          <MenuItem className={classes.menuItem}>
+            <Button component={Link} to="/signup" variant="contained" color="primary" className={classes.signUpBtn}>
+              SIGN UP
             </Button>
-          </div>
-          <Button component={Link} to="/signup" variant="contained" color="secondary" className={classes.signUpBtn}>
-            SIGN UP
-          </Button>
-        </Toolbar>
-      </AppBar>
-    </div>
+          </MenuItem>
+        </MenuList>
+      </div>
+    </Toolbar>
+  );
+
+  const displayMobile = () => {
+    const handleDrawerOpen = () => setIsDrawerOpen(true);
+    const handleDrawerClose = () => setIsDrawerOpen(false);
+    return (
+      <Toolbar className={mobileToolbar}>
+        <IconButton className={mobileIcon} onClick={handleDrawerOpen}>
+          <MenuIcon />
+        </IconButton>
+
+        <Drawer anchor="right" open={isDrawerOpen} onClose={handleDrawerClose}>
+          <div className={drawerContainer}>{getDrawserChoices()}</div>
+        </Drawer>
+
+        <Link to="/">
+          <img src={Logo} className={logo} alt="logo" />
+        </Link>
+      </Toolbar>
+    );
+  };
+
+  const getDrawserChoices = () => {
+    const handleNavMenuClick = () => {
+      setIsDrawerOpen(false);
+    };
```

Comment:
This is being repeated in this file, I would move this close handlers to the LandingNavbar component and just utilize that accordingly as opposed to defining multiple function expressions that do the same thing.

Reply:
Sure, will do that.